money_bot/handlers/earn_handlers.py

```diff
@@ -0,0 +1,76 @@
+from aiogram import Dispatcher, types
+
+from money_bot.initialization import bot
+from money_bot.utils import db_utils, markups
+from money_bot.utils.states import GlobalStates
+from money_bot.utils.strings import EARN_MENU_TEXT, config
+
+
+async def entry_point(message: types.Message, new=False, last_message=None, user_id=None):
```

Comment:
Add some docs about what `new` and `last_message` means, as their meanings aren't straightforward